DepositeCalcTests/Tests/CalculatorPageTests.cs

```diff
@@ -0,0 +1,268 @@
+﻿using DepositeCalcTests.Pages;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+
+namespace DepositeCalcTests.Tests
+{
+    internal class CalculatorPageTests
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void Setup()
+        {
+            ChromeOptions options = new ChromeOptions { AcceptInsecureCertificates = true };
+            driver = new ChromeDriver(options);
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(30);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5);
+            driver.Url = "https://localhost:5001/Calculator";
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
+
+        [TestCase("", "", "")]
+        [TestCase("", "", "1")]
+        [TestCase("", "1", "")]
+        [TestCase("", "1", "1")]
+        [TestCase("1", "", "")]
+        [TestCase("1", "", "1")]
+        [TestCase("1", "1", "")]
```

Comment:
I think these three cases should be enough:
```suggestion
        [TestCase("", "1", "1")]
        [TestCase("1", "", "1")]
        [TestCase("1", "1", "")]
```